app/views/hosts/_templates.html.erb

```diff
@@ -1,3 +1,6 @@
+<%
+  can_preview = authorized_for(:permission => :view_provisioning_templates, :auth_object => template, :authorizer => authorizer)
```

Comment:
This should be inside of `<% @templates.each do |template| %>` block

Reply:
oh. yes. for sure :)